pac/docker-build-rhtap/docker-pull-request.yaml

```diff
@@ -36,6 +36,9 @@ spec:
   pipelineRef:
     name: docker-build-rhtap
   workspaces:
+    - name: git-auth
```

Comment:
`git-auth` is referenced in the docker-build-rhtap pipeline here: https://github.com/redhat-appstudio/tssc-sample-pipelines/blob/main/pac/pipelines/docker-build-rhtap.yaml#L119-L120